packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -458,6 +458,12 @@ class _RangeSliderState extends State<RangeSlider> with TickerProviderStateMixin
     }
   }
 
+  // Always keep the ValueIndicator visible on the Overlay; otherwise, it cannot be updated during the build phase.
+  late final OverlayPortalController _valueIndicatorOverlayPortalController =
```

Comment:
the late is probably unnecessary as the controller will 100% ( I think ) be evaluated later in `build`. Might as well just construct it right away?